<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>624</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="verticalSpacing">
        <number>0</number>
       </property>
       <item row="1" column="3" colspan="2">
        <widget class="QDateEdit" name="dateEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="6" colspan="2">
        <widget class="QTimeEdit" name="timeEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_date_gan_zhi">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>客户姓名</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" rowspan="2">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>日期</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="label_hour_gan_zhi">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="5" rowspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>时间</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>调理方案</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButton_NaJia">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>
纳甲法
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_NaZi">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>
纳子法
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_LingGui8">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>
灵龟八法
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_FeiTeng8">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>
飞腾八法
</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetMinimumSize</enum>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="sizeConstraint">
          <enum>QLayout::SizeConstraint::SetFixedSize</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>针刺顺序 </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_order">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>男左女右</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>巨刺法</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="sizeConstraint">
            <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>时穴类型 </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_type">
             <property name="minimumSize">
              <size>
               <width>130</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
             <item>
              <property name="text">
               <string>就诊时穴</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>精神心理刺激</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>意外事故</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>错误或过度治疗</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>病情加重的事件</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>出生时间</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="verticalWidget_choose_acupoint" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_a">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_b">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_c">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_d">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_e">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_f">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>穴位</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="radioButton_a">
              <property name="text">
               <string>穴位a</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_b">
              <property name="text">
               <string>穴位b</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_c">
              <property name="text">
               <string>穴位c</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_d">
              <property name="text">
               <string>穴位d</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_e">
              <property name="text">
               <string>穴位e</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_f">
              <property name="text">
               <string>穴位f</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>生成
方案</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton_NaJia</sender>
   <signal>clicked()</signal>
   <receiver>label_e</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaJia</sender>
   <signal>clicked()</signal>
   <receiver>label_f</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaJia</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_e</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>560</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaJia</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_f</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaZi</sender>
   <signal>clicked()</signal>
   <receiver>label_e</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaZi</sender>
   <signal>clicked()</signal>
   <receiver>label_f</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaZi</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_e</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>261</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaZi</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_f</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>659</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_LingGui8</sender>
   <signal>clicked()</signal>
   <receiver>verticalWidget_choose_acupoint</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_FeiTeng8</sender>
   <signal>clicked()</signal>
   <receiver>verticalWidget_choose_acupoint</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>488</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaJia</sender>
   <signal>clicked()</signal>
   <receiver>verticalWidget_choose_acupoint</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_NaZi</sender>
   <signal>clicked()</signal>
   <receiver>verticalWidget_choose_acupoint</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
